File: 509_fibonacci_number/fib_num.py
# Py3: https://leetcode.com/problems/fibonacci-number/submissions/1212935425/
def fib(n):

    def rec(n):
        if n == 0:
            return 0
        if n == 1:
            return 1
        
    return rec(n - 1) + rec(n - 2)

print(fib(2))

File: 509_fibonacci_number/fib_num.js
// JS: https://leetcode.com/problems/fibonacci-number/submissions/1212936428/
// TS: https://leetcode.com/problems/fibonacci-number/submissions/1212936012/
function fib(n) {
    if(n === 0) return 0;
    if(n === 1) return 1;
    
    return fib(n - 1) + fib(n - 2);
}

console.log(fib(2))